gears/src/types/context/init.rs

```diff
@@ -70,4 +74,8 @@ impl<DB: Database, SK: StoreKey> TransactionalContext<DB, SK> for InitContext<'_
     fn events_drain(&mut self) -> Vec<Event> {
         std::mem::take(&mut self.events)
     }
+
+    fn get_time(&self) -> Option<Timestamp> {
```

Comment:
This should not be optional

Reply:
I added the trait member as we discussed before and added the comment here https://github.com/rumos-io/gears/pull/97/files#diff-fa0d5bf00d58a58c105692976c77d149771eabf0f905bb77355e791c5e0a198dR27. 

Should I change the realization of `Header` in this PR?